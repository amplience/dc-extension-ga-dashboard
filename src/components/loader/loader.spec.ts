import { render } from '@testing-library/svelte';
import Loader from './loader.svelte';

describe('Loader', () => {
  it('should render the Loader component', () => {
    const { container } = render(Loader, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
