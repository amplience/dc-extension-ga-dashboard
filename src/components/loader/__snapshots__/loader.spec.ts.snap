// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render the Loader component 1`] = `
<div>
  <div
    class="spinner-container svelte-bvd1nw"
  >
    <div
      class="spinner dot-0 svelte-bvd1nw"
    />
    <div
      class="spinner dot-1 svelte-bvd1nw"
    />
    <div
      class="spinner dot-2 svelte-bvd1nw"
    />
  </div>
</div>
`;
